stdlib/json.gr

```diff
@@ -0,0 +1,2157 @@
+/**
+ * JSON (JavaScript Object Notation) parsing and printing.
+ */
+
+module Json
+
+include "string"
+include "char"
+include "list"
+include "option"
+include "number"
+include "uint8"
+include "runtime/unsafe/wasmi32"
+include "runtime/unsafe/wasmi64"
+include "runtime/unsafe/wasmf32"
+include "runtime/unsafe/wasmf64"
+include "runtime/numberUtils"
+include "runtime/unsafe/tags"
+include "runtime/numbers"
+include "runtime/unsafe/memory"
```

Comment:
This is mostly a matter of personal preference, but I prefer an `include` section for runtime includes, followed by a section for regular stdlib includes, with both sorted alphabetically.